plugins/automation/admin/assets/javascripts/admin/components/fields/da-base-field.gjs

```diff
@@ -2,6 +2,14 @@ import Component from "@glimmer/component";
 import { action } from "@ember/object";
 
 export default class BaseField extends Component {
+  constructor() {
+    super(...arguments);
+
+    if (this.args.field.extra?.default_value) {
```

Comment:
if the default value is `false` here this won't work, right? I think you need to do something like 
```js
if (this.args.field.extra && Object.keys(this.args.field.extra).includes("default_value")
```